<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Balance</class>
 <widget class="QWidget" name="Balance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1435</width>
    <height>788</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HP Simplified</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelBalance">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>46</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N채yt채 saldo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBalanceDate">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>08.04.2019 11:00:12 </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTextBalance">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tilin saldo</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowBalance">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>430</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>90</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReturnMenu">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>290</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>P채채valikko</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
